pilot/pkg/credentials/kube/secrets.go

```diff
@@ -37,28 +37,6 @@ import (
 	"istio.io/istio/pkg/log"
 )
 
-const (
-	// The ID/name for the certificate chain in kubernetes generic secret.
-	GenericScrtCert = "cert"
-	// The ID/name for the private key in kubernetes generic secret.
-	GenericScrtKey = "key"
-	// The ID/name for the CA certificate in kubernetes generic secret.
-	GenericScrtCaCert = "cacert"
-	// The ID/name for the CRL in kubernetes generic secret.
-	GenericScrtCRL = "crl"
-
-	// The ID/name for the certificate chain in kubernetes tls secret.
-	TLSSecretCert = "tls.crt"
-	// The ID/name for the k8sKey in kubernetes tls secret.
-	TLSSecretKey = "tls.key"
-	// The ID/name for the certificate OCSP staple in kubernetes tls secret
-	TLSSecretOcspStaple = "tls.ocsp-staple"
-	// The ID/name for the CA certificate in kubernetes tls secret
-	TLSSecretCaCert = "ca.crt"
-	// The ID/name for the CRL in kubernetes tls secret.
-	TLSSecretCrl = "ca.crl"
-)
-
```

Comment:
Have moved these to `pilot/pkg/credentials`, if not supposed to be there, we can also move to `pilot/pkg/credentials/kube/model`